### PR TITLE
update content and styling

### PR DESCRIPTION
**Pull request by stephenBarrs**
## Description
Update to content and styling. Include Height form field. 

## Testing done
Manual Testing

## Acceptance criteria
- [x] height form element is displayed 
- [x] updated content is displayed

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/covid-research-volunteer/pages/covidResearchUISchema.js [hunk 1]
```diff
@@ -19,15 +19,19 @@ export const uiSchema = {
...
+          volunteer registry. If we think you may be eligible for one of our
+          COVID-19 studies, we’ll contact you to tell you more about it so you
+          can decide if you want to join. You don’t need to be a Veteran to
+          volunteer.
           <p>
             <b>Note:</b> We won’t share your information with anyone outside of
-            VA. To learn more before volunteering, read about frequently asked
-            questions about COVID-19 research at VA.
+            VA. To learn more before volunteering, read about{' '}
+            <a href="https://staging.va.gov/alt-covid-research-lp/">
```
Comment (mchelen-gov) [inline src/applications/covid-research-volunteer/pages/covidResearchUISchema.js:31]:
```suggestion
            <a href="/alt-covid-research-lp">
```
use relative link so it is independent from environment

File: src/applications/covid-research-volunteer/containers/ConfirmationPage.jsx [hunk 2]
```diff
@@ -55,6 +57,9 @@ export class ConfirmationPage extends React.Component {
             We won’t contact you. But we’ll keep your information on file for
             future studies you may be eligible to join.
           </p>
+          <a href="preview-prod.vfs.va.gov/preview?nodeId=6785">
```
Comment (mchelen-gov) [inline src/applications/covid-research-volunteer/containers/ConfirmationPage.jsx:60]: is this supposed to be the same as the other link?

File: src/applications/covid-research-volunteer/pages/covidResearchUISchema.js [hunk 3]
```diff
@@ -259,11 +266,12 @@ export const uiSchema = {
           On most days, how many people do you have close contact with outside
           of those who live in your home?
         </strong>
+        <br />
+        <br />
+        <strong>Note:</strong> We define close contact as being within 6 feet of
+        a person.
```
Comment (mchelen-gov) [inline src/applications/covid-research-volunteer/pages/covidResearchUISchema.js:272]:
```suggestion
        <p>
          <strong>Note:</strong> We define close contact as being within 6 feet
          of a person.
        </p>
```
might be better way to achieve desired spacing

File: src/applications/covid-research-volunteer/sass/covid-research-volunteer.scss [hunk 4]
```diff
@@ -6,3 +6,8 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../platform/forms/sass/m-form-confirmation";
+
+.input-width {
+  display: inline-block;
+  max-width: 20rem;
```
Comment (mchelen-gov) [inline src/applications/covid-research-volunteer/sass/covid-research-volunteer.scss:12]:
```suggestion
  max-width: units(15);
```
saw this recently on https://design.va.gov/design/spacing-units#using-the-spacing-tokens, not sure if it's helpful here or not

**Review by mchelen-gov — CHANGES_REQUESTED** [review]
url fixes

File: src/applications/covid-research-volunteer/containers/ConfirmationPage.jsx (hunk 2 above)
Comment (stephenBarrs) [reply src/applications/covid-research-volunteer/containers/ConfirmationPage.jsx:60]: Need to confirm these links with content folks

Comment (stephenBarrs) [reply src/applications/covid-research-volunteer/containers/ConfirmationPage.jsx:60]: These links are supposed to be the same, currently planned for `/coronavirus-research`. updated locally and will push up.

**Comment by stephenBarrs** [comment]
@mchelen-gov can you re-review this with changes?  Thanks!

**Review by mchelen-gov — APPROVED** [review]

**Pull request merged**